docs/src/howto-guides/026_how_to_install_thin_edge_manually.md

```diff
@@ -0,0 +1,224 @@
+# How to install thin-edge.io manually with OpenRC
+
+This tutorial will demo how to install thin-edge.io manually for a non-debian linux distribution that uses OpenRC as its init system.
+The aim of this tutorial is to show how to get started with Cumulocity IoT even if your current system is not supported by the default installation of thin-edge.io.
+For reference, this tutorial is done with the following system specs:
+
+- Operating System: Linux gentoo
+- Linux kernel version: 5.15.41-gentoo-x86\_64
+- Architecture: x86\_64
+- Init system: OpenRC
+
+## Prerequisites
+
+If you wish to [build binaries from source](#building-from-source), you will to install rust from https://www.rust-lang.org/tools/install.
+
+You will also need to have mosquitto installed. Check your package manager for an available version, or you can [building from source](https://github.com/eclipse/mosquitto). (If you build from source, add `WITH_TLS=yes` flag to make).
```

Comment:
Since you provide Gentoo as an example below, I would add install information for Gentoo here.